app/components/ui/molecules/DashboardListItem.js

```diff
@@ -29,6 +29,13 @@ const dashboardDateText = date => {
   return `${diffDays} ${diffDays === 1 ? 'day' : 'days'} ago`
 }
 
+const mapColor = statusCode =>
```

Comment:
Perhaps this is a good example of why `ManuscriptStatus` should be part of `DashboardListItem`? We now have the colour mapping function being repeated inside 2 files.

I'm all for breaking components into small pieces, but only when the smallest component makes sense in isolation. In this situation, we don't use `ManuscriptStatus` anywhere else & it actually adds complexity having these in separate files, rather than reducing complexity.

@jyenes @Cooryd Any thoughts?